docs/src/components/copy-command-button.tsx

```diff
@@ -0,0 +1,68 @@
+'use client';
+
+import { useState } from 'react';
+import { copyTextToClipboard } from '../utils';
+import Button from './button';
+
+export function CopyCommandButton() {
+  const copyCommandText = 'npm create @keystatic@latest';
+  const [isCopied, setIsCopied] = useState(false);
+
+  const handleCopyCommand = () => {
+    copyTextToClipboard(copyCommandText)
+      .then(() => {
+        setIsCopied(true);
+        setTimeout(() => {
+          setIsCopied(false);
+        }, 1500);
+      })
+      .catch(err => {
+        console.log(err);
+      });
+  };
```

Comment:
We should be able to convert this to async/await like so (I haven't tested this!). Bit easier on the eyes then promises.

```suggestion
  const handleCopyCommand = async () => {
    try {
      await copyTextToClipboard(copyCommandText);
      setIsCopied(true);
      setTimeout(() => {
        setIsCopied(false);
      }, 1500);
    }
    catch(err) {
      console.log(err);
    }
  };
```